inception-project/inception
# #825 external search bar - keyword highlight

#825 External Search Side bar - Keyword Highlight

**What's in the PR**
This is an additional feature in the #825 to add keyword highlight to annotation page

**How to test manually**
1. Select a project and go to Annotation section from the project dashboard.
2. From the sidebar, select external search.
3. Do a search in the external search.
4. Click on a result.
5. When the annotation appears, there should be keywords highlighting in the annotation page.
6. The keywords highlighted should match the same sentence and same keyword highlight in the result row.

**Automatic testing**
None
